README.md

```diff
@@ -42,7 +42,34 @@ Result:
 ![none](https://bots.are-pretty.sexy/4eae08.png)
 
 ## Client ID
-TODO: Document on how to make your own application and upload an asset
+It's relatively simple to make your own application to use for RPC. First of all, head to https://discordapp.com/developers/applications/me and click on the "New App" button.
+![none](https://mikecao.me/i/2026c7.png)
+
+Next, you'll want to give your application a title and upload a picture for it if you want.
```

Comment:
I'd note the naming shouldn't be a verified program like "Spotify" to avoid the blacklist